agent/exec/containerd/executor.go

```diff
@@ -0,0 +1,107 @@
+package containerd
```

Comment:
> Perhaps we should consider renaming the former to `github.com/docker/swarmkit/agent/exec/dockerengine` (or `.../dockerapi` or whatever suites)?

I agree. I find the current naming confusing. I think either `dockerengine` or `dockerapi` is better. I guess I have a slight preference for `dockerapi`.